src/pages/returnDirection.tsx

```diff
@@ -118,11 +110,11 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{}> => {
     }
 
     // Redirect to inputMethod page if there is only one journeyPattern (i.e. circular journey)
-    if (service.journeyPatterns.length === 1 && fareTypeInfo.fareType === 'returnSingle') {
+    if (service.journeyPatterns.length === 1 && fareTypeInfo.fareType === 'return') {
         if (ctx.res) {
             const uuid = getUuidFromCookies(ctx);
             const journeyPatternCookie = `${service.journeyPatterns[0].startPoint.Id}#${service.journeyPatterns[0].endPoint.Id}`;
-            const cookieValue = JSON.stringify({ journeyPattern: journeyPatternCookie, uuid });
+            const cookieValue = JSON.stringify({ directionJourneyPattern: journeyPatternCookie, uuid });
```

Comment:
This is going to route circular journeys through the single matching page which is fine but it will end up setting the type as `pointToPoint` instead of `return`, we'll need some logic on the matching.ts api to check the type and set it correctly. We should also make sure that the fare zones in the matching json have a key of `outboundFareZones` when the fare type is return and `inboundFareZones` should be an empty array to make it easier for the NeTEx convertor